### PR TITLE
support asyncpg 0.22.0

### PR DESCRIPTION
**Pull request by Gr1N**
Hi!

The new release of `asyncpg` 0.22.0 breaks compatibility with `asyncpgsa` and here is a simple fix for that. Please review and let me know if I need to fix something.

Thanks!

### CODE REVIEW

**Comment by Ragnaruk** [comment]
I believe _execute should look like this:

```
    def _execute(
        self,
        query,
        args,
        limit,
        timeout,
        *,
        return_status=False,
        ignore_custom_codec=False,
        record_class=None
    ):
        query, compiled_args = compile_query(query, dialect=self._dialect)
        args = compiled_args or args
        return super()._execute(
            query,
            args,
            limit,
            timeout,
            return_status=return_status,
            record_class=record_class,
            ignore_custom_codec=ignore_custom_codec,
        )
```

Did your version work?

**Comment by Gr1N** [comment]
@Ragnaruk I ran tests with my fix and they passed

File: asyncpgsa/connection.py [hunk 1]
```diff
@@ -88,11 +88,12 @@ def __init__(self, *args, dialect=None, **kwargs):
...
         self._dialect = dialect or _dialect
 
-    def _execute(self, query, args, limit, timeout, return_status=False):
+    def _execute(self, query, args, limit, timeout, return_status=False, record_class=None):
         query, compiled_args = compile_query(query, dialect=self._dialect)
         args = compiled_args or args
         return super()._execute(query, args, limit, timeout,
-                                return_status=return_status)
+                                return_status=return_status,
+                                record_class=record_class)
```
Comment (FreddieLindsey) [inline asyncpgsa/connection.py:96]:
```suggestion
                                record_class=record_class,
                                ignore_custom_codec=ignore_custom_codec)
```

File: asyncpgsa/connection.py [hunk 2]
```diff
@@ -88,11 +88,12 @@ def __init__(self, *args, dialect=None, **kwargs):
         super().__init__(*args, **kwargs)
         self._dialect = dialect or _dialect
 
-    def _execute(self, query, args, limit, timeout, return_status=False):
+    def _execute(self, query, args, limit, timeout, return_status=False, record_class=None):
```
Comment (FreddieLindsey) [inline asyncpgsa/connection.py:91]:
```suggestion
    def _execute(self, query, args, limit, timeout, return_status=False, record_class=None, ignore_custom_codec=False):
```

**Review by nhumrich — APPROVED** [review]
Thank you, this is awesome. Good improvements to the build process as well!

**Comment by nhumrich** [comment]
This is included in asyncpgsa 0.27.0

**Pull request merged**